src/psfmachine/tpf.py

```diff
@@ -1088,10 +1088,11 @@ def _parse_TPFs(tpfs, renormalize_tpf_bkg=True, **kwargs):
     sat_mask = []
     for tpf in tpfs:
         # Keplerish saturation limit
-        saturated = np.nanmax(tpf.flux, axis=0).T.value > 1.4e5
-        saturated = np.hstack(
-            (np.gradient(saturated.astype(float))[1] != 0) | saturated
-        )
+        saturated = np.nanmax(tpf.flux, axis=0).T.value > 1.2e5
```

Comment:
This is hardcoded. I think when we switch this up for TESS it'll make things a pain. Let's leave it for now and make sure when we refactor we catch these